lib/cli/ui/spinner/spin_group.rb

```diff
@@ -475,6 +475,13 @@ def all_succeeded?
           end
         end
 
+        sig { returns(T::Array[Exception]) }
+        def all_exceptions
+          @m.synchronize do
+            @tasks.filter_map(&:exception)
+          end
+        end
+
```

Comment:
I think changes to cli-ui should be made on that repo, and then we bump the vendored version here